src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -108,35 +108,23 @@ protected override async Task HandleWebResourceRequest(CoreWebView2WebResourceRe
 					if (winUIItem != null)
 					{
 						using var contentStream = await Package.Current.InstalledLocation.OpenStreamForReadAsync(relativePath);
-
-						var memStream = new MemoryStream();
-						contentStream.CopyTo(memStream);
-						stream = new InMemoryRandomAccessStream();
-						await stream.WriteAsync(memStream.GetWindowsRuntimeBuffer());
+						stream = await CopyContentToRandomAccessStreamAsync(contentStream);
 					}
 				}
 				else
 				{
 					var path = Path.Combine(AppContext.BaseDirectory, relativePath);
 					if (File.Exists(path))
 					{
-						// NOTE: This is stream copying is to work around a hanging bug in WinRT with managed streams.
-						// See issue https://github.com/microsoft/CsWinRT/issues/670
```

Comment:
No longer true; https://github.com/dotnet/maui/pull/9254

We've kept the copy to work around some file locking access issues (I'll create a new issue and add a comment).